src/Api/index.js

```diff
@@ -0,0 +1,63 @@
+import axios from 'axios';
+
+
+const instance = axios.create({
+    baseURL: 'http://localhost:3000'
+})
+
+
+instance.interceptors.request.use(function (config) {
+    console.log('Request sent successfully');
+    return config;
+}, function (err) {
+    console.log(err.status)
+    return Promise.reject(err);
+})
+
+instance.interceptors.response.use(function (response) {
+    console.log(response.status);
+    return response;
+}, function (err) {
+    console.log(err.message);
+    return Promise.reject(err);
+})
```

Comment:
Refactor Advice: create 4 functions. outgoingRequest(Success|Failure)Interceptor and incomingResponse(Success|Failure)Interceptor and use them like:
`instance.interceptors.request.use(outgoingRequestSuccessInterceptor,outgoingRequestFailureInterceptor)`